packages/hardhat/contracts/FundingVault.sol

```diff
@@ -56,13 +57,34 @@ contract FundingVault{
     uint256 private blockLimit;
     uint256 private minFundingAmount; //The minimum amount of ETH required in the contract to enable withdrawal.
     uint256 private exchangeRate; //The exchange rate of ERG per token
+    address private projectOwner;
     address private withdrawlAddress;
     address private developerFeeAddress; //develper address
     uint256 private developerFeePercentage; 
     string  private projectURL;
+    string private projectTitle;
+    string private projectDescription;
     bool private fundsWithdrawn;
 
 
+    /** 
+    * @dev A vault is represented as a struct  
+    */ 
+
+    struct Vault{
```

Comment:
We don't need this struct.

Reply:
Suggestion Required: This struct is used for function getVauts() Having a single function rather than multiple public variables reduces the number of RPC call to just one.
Otherwise we need multiple contract calls to fetch each variable.
Isn't it desirable? Please help with this.